examples/Atlas/dev/runAtlasWalkingStairs.m

```diff
@@ -7,10 +7,12 @@ function runAtlasWalkingStairs()
 path_handle = addpathTemporary(fullfile(getDrakePath(), 'examples', 'Atlas'));
 
 robot_options = struct();
-robot_options = applyDefaults(robot_options, struct('use_bullet', false,...
+robot_options = applyDefaults(robot_options, struct('use_bullet', true,...
                                                     'terrain', RigidBodyFlatTerrain,...
                                                     'floating', true,...
                                                     'ignore_self_collisions', true,...
+                                                    'ignore_effort_limits', true,...
+                                                    'enable_fastqp', true,...
```

Comment:
yeah, for the v3 legs. I think v5 will be better, so we should be able to turn this off once it's in drake